### PR TITLE
findRoot: fix Issue 14231

### PR DESCRIPTION
**Pull request by 9il**
https://issues.dlang.org/show_bug.cgi?id=14231

### CODE REVIEW

**Comment by e10s** [comment]
Looks nice.

BTW, considering the difference between `-0.0` and `+0.0`, the ddoc seems to say this function still requires `f(a) <= -0 && +0 <= f(b) || f(b) <= -0 && +0 <= f(a)`, which has been implicitly contracted. Now that it allows `(f(a) == 0 || f(b) == 0) && ( f(a) <= -0 && f(b)  <= -0 || +0 <= f(a) && +0 <= f(b) )` as well, you could modify the description.

**Comment by 9il** [comment]
Thanks! Docs was fixed.

**Comment by MartinNowak** [comment]
Auto-merge toggled on

**Comment by MartinNowak** [comment]
thx

File: std/numeric.d [hunk 1]
```diff
@@ -838,8 +845,8 @@ T findRoot(T, DF)(scope DF f, in T a, in T b)
  *
  * fax = Value of $(D f(ax)).
  *
- * fbx = Value of $(D f(bx)). ($(D f(ax)) and $(D f(bx)) are commonly
- * known in advance.)
+ * fbx = Value of $(D f(bx)). (D fax) and $(D fbx) should have opposite signs.
```
Comment (don-clugston-sociomantic) [inline std/numeric.d]: Missing dollar sign. That should be ``$(D fax)``.

Comment (9il) [inline std/numeric.d]: Fixed in https://github.com/D-Programming-Language/phobos/pull/3084

**Pull request merged**